humans.txt

```diff
@@ -0,0 +1,32 @@
+/* TEAM */
+
+	Chef: Jaydson Gomes
+	Contact: jayalemao@gmail.com
+	Website: http://jaydson.org
+	Twitter: @jaydson
+	From: Porto Alegre
+
+	Web designer: Ítalo Waxman
+	Contact: http://br.linkedin.com/in/italowaxman/
+	Twitter: @italowaxman
+	From: Aracajú
+
+	Theme developer: Robson Júnior
+	Contact: robson.ga.junior@gmail.com
+	Website: http://robsonjunior.cc
+	Twitter: @robsongajunior
+	From: Porto Alegre
+
+
+/* THANKS */
```

Comment:
I believe the "Thanks" section is for people who contributed but are not really part of the team. Can you remove the "Thanks" section and move my data to below Jaydson?
Also, please add `Website: http://ultcombo.js.org`